unfoldingWord/translationCore
# Fixed user page instructions based on QA

#### This pull request addresses:
```
Changed Instructions to say:

You are currently logged in.

To continue to Projects, click "Continue to Project".

To log out, click "Log out".
```
```
 The verbiage on the main card  should be:
You are logged in as:
[username]
How would you like to proceed?
```

#### How to test this pull request:

- Open the app and Verify the text has been changed


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2107)
